app/components/Form/SelectInput.tsx

```diff
@@ -80,7 +80,8 @@ const NO_OPTIONS_MESSAGE = 'Ingen treff';
 const LOADING_MESSAGE = 'Laster inn ...';
 
 const SelectInput = <
-  Option extends { label: string; value: number },
+  T,
+  Option extends { label: string; value: T },
```

Comment:
is this used?

Reply:
It's maybe not the best solution, but it fixes typescript errors when `value` is not a `number` (the most common case afaik). I assume `value: any` would not be better?